src/hacbs/components/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Dropdown, DropdownItem, KebabToggle } from '@patternfly/react-core';
+import { RowFunctionArgs, TableData } from '../../../shared/components/table';
+import { Timestamp } from '../../../shared/components/timestamp/Timestamp';
+import { PipelineRunKind } from '../../types';
+import { calculateDuration } from '../../utils/pipeline-utils';
+import { pipelineRunTableColumnClasses } from './PipelineRunListHeader';
+
+const PipelineListRow: React.FC<RowFunctionArgs<PipelineRunKind>> = ({ obj }) => {
+  const [kebabOpen, setKebabOpen] = useState(false);
+
+  const capitalize = (label: string) => {
+    return label.charAt(0).toUpperCase() + label.slice(1);
+  };
+  return (
+    <>
+      <TableData className={pipelineRunTableColumnClasses.name}>
+        <Link to={`/app-studio/pipelineruns?name=${obj.metadata.name}`} title={obj.metadata.name}>
+          {obj.metadata.name}
+        </Link>
+      </TableData>
+      <TableData className={pipelineRunTableColumnClasses.started}>
+        <Timestamp
+          timestamp={typeof obj.status.startTime === 'string' ? obj.status.startTime : ''}
```

Comment:
`obj.status` could be `undefined` ?

Reply:
checked

Reply:
There are more places that require this check.